JavaScripit/index.js

```diff
@@ -0,0 +1,28 @@
+var today = new Date();
+var thisYear = today.getFullYear();
+var footer = document.querySelector("footer");
+var copyright = document.createElement('p');
+
+copyright.innerHTML = "Haregebezu Amare; " + thisYear;
+
+footer.appendChild(copyright);
+copyright.innerHTML = "\u00A9 " + thisYear;
```

Comment:
This clobbers the value you set above: Clarify what the footer should contain, and make sure you have no extraneous code.